src/main/kotlin/gg/skytils/skytilsmod/features/impl/misc/ItemFeatures.kt

```diff
@@ -414,6 +415,27 @@ object ItemFeatures {
                 }"
             })
         }
+
+        if (Skytils.config.showItemQuality && extraAttr != null) {
+            val boost = extraAttr.getInteger("baseStatBoostPercentage")
+            val tier = extraAttr.getInteger("item_tier")
+
+            if (boost > 0 && tier > 0) {
+                val isMasterMode = requirementPattern.matchEntire(extraAttr.getString("dungeon_skill_req"))?.let {
+                    val level = it.groups["level"]?.value?.toIntOrNull() ?: return@let null
```

Comment:
You can directly use `MatchResult#groupValues` here to avoid getting the individual group.